libs/utils/android/benchmark.py

```diff
@@ -116,7 +116,7 @@ def _parseCommandLine(self):
 
         # Test customization
         parser.add_argument('--results-dir', type=str,
-                default=None,
+                default=self.__class__.__name__,
```

Comment:
Well... the current implementation uses the path specified by `--results-dir`  that why:
- relative path: append it to the standard `results` dir you references
- absolute path: use whatever you specified

Thus, in the case of the Joel's proposal, the class name would be interpreted as a "relative path" and we will end up with the results under `$LISA_HOME/results/<CLASS_NAME>`... which seems a sensible default to me.